<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueInfoPanelClass</class>
 <widget class="QWidget" name="IssueInfoPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>581</width>
     <height>581</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>490</width>
     <height>491</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-711</y>
      <width>562</width>
      <height>1300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1300</height>
     </size>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>551</width>
       <height>1081</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="14" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>記錄人:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>四環:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" colspan="3">
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>四環分類:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>問題描述:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" colspan="3">
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QComboBox" name="VehicleModel_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>真因分類:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="Program_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>車端軟體版本:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QComboBox" name="IssueType_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>類型:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>記錄日期:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>嚴重等級:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>車廠:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>車型:</string>
        </property>
       </widget>
      </item>
      <item row="20" column="4">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="18" column="2" colspan="8">
       <widget class="QTextEdit" name="Solution_textEdit"/>
      </item>
      <item row="17" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>问题原因:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <spacer name="horizontalSpacer_16">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>APP版本:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6" colspan="3">
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="6">
       <spacer name="horizontalSpacer_17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="20" column="2">
       <widget class="CDateEdit" name="Estimate_dateEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QComboBox" name="Supplier_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="21" column="4">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="16" column="2" colspan="3">
       <widget class="QComboBox" name="State_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="6">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="5">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2" colspan="4">
       <widget class="QLineEdit" name="ID_lineEdit"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>項目:</string>
        </property>
       </widget>
      </item>
      <item row="21" column="2">
       <widget class="CDateEdit" name="Actual_dateEdit">
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="8">
       <widget class="QTextEdit" name="IssueDescribe_textEdit"/>
      </item>
      <item row="18" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>解决方案:</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>狀態:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>負責人:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>安全等级:</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>改善狀況描述:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6" colspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="12" column="2" colspan="3">
       <widget class="QComboBox" name="Reason_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="14" column="2" colspan="3">
       <widget class="QLineEdit" name="RecordPerson_lineEdit"/>
      </item>
      <item row="13" column="2" colspan="3">
       <widget class="QComboBox" name="SafeLevel_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="5">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="2" colspan="3">
       <widget class="QComboBox" name="FourRingType_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="15" column="2" colspan="3">
       <widget class="QLineEdit" name="PersonInCharge_lineEdit"/>
      </item>
      <item row="7" column="2" colspan="3">
       <widget class="QLineEdit" name="VehicleVersion_lineEdit"/>
      </item>
      <item row="13" column="5">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="2">
       <widget class="QComboBox" name="Level_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="2" colspan="3">
       <widget class="QComboBox" name="FourRing_comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>预计完成日期:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDateEdit" name="Record_dateEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2020</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="11" column="6">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="17" column="2" colspan="8">
       <widget class="QTextEdit" name="IssueResson_textEdit"/>
      </item>
      <item row="15" column="5">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="2" colspan="3">
       <widget class="QLineEdit" name="APPVersion_lineEdit"/>
      </item>
      <item row="16" column="5">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="19" column="2" colspan="8">
       <widget class="QTextEdit" name="Result_textEdit"/>
      </item>
      <item row="21" column="0" colspan="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>实际完成日期:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="6">
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="20" column="3">
       <widget class="QPushButton" name="CleanEstimateDateBtn">
        <property name="text">
         <string>清除日期</string>
        </property>
       </widget>
      </item>
      <item row="21" column="3">
       <widget class="QPushButton" name="CleanActualDateBtn">
        <property name="text">
         <string>清除日期</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">date_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
